lib/money/money/formatting.rb

```diff
@@ -345,7 +345,7 @@ def regexp_format(formatted, rules, decimal_mark, symbol_value)
 
   def localize_formatting_rules(rules)
     if currency.iso_code == "JPY" && I18n.locale == :ja
-      rules[:symbol] = "円" unless rules[:symbol] == false
+      rules[:symbol] = "円" if rules[:symbol] == true
```

Comment:
I would think that we'd want `money.format(:symbol => '¥')` to work, and should add a spec for that.

We do already have a spec for `expect(money.format(:symbol => false)).to eq "1,000"` which is the case in which `if rules[:symbol]` would not suffice.